cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -497,6 +503,60 @@ def get_schedule(
         filter_str = ' AND '.join(filters)
         return self.context.client.list_time_slots(self.project_id, self.processor_id, filter_str)
 
+    def get_config_from_run(
+        self, run_name: str = 'default', config_alias: str = 'default'
+    ) -> processor_config.ProcessorConfig | None:
+        """Retrieves a ProcessorConfig from an automation run.
+
+            If no run name is provided, the config from the most recent run
+            is returned.
+
+        Args:
+            processor_id: The processor unique identifier.
+            config_alias: The quantum processor's unique identifier.
+            run_name: The automation run name.  Use 'default'
+                      if none id provided.
+
+        Returns: The quantum processor config.
+        """
+        response = self.context.client.get_quantum_processor_config_from_run(
```

Comment:
WDYT of forwarding to the Engine implementation to ensure that both versions implement the same functionality, e.g. what to call, how to transform the response? e.g., that's also done for `run_sweep`.